bundle/metadata/annotations.yaml

```diff
@@ -1,7 +1,15 @@
 annotations:
-  operators.operatorframework.io.bundle.channel.default.v1: stable
-  operators.operatorframework.io.bundle.channels.v1: stable,stable-5.4
-  operators.operatorframework.io.bundle.manifests.v1: manifests/
+  # Core bundle annotations.
   operators.operatorframework.io.bundle.mediatype.v1: registry+v1
+  operators.operatorframework.io.bundle.manifests.v1: manifests/
   operators.operatorframework.io.bundle.metadata.v1: metadata/
-  operators.operatorframework.io.bundle.package.v1: cluster-logging
+  operators.operatorframework.io.bundle.package.v1: cluster-logging-operator
```

Comment:
I think all these changes needed to be reverted given the package changed

Reply:
@vparfonov other then either: fixing the package name for this file or reverting it entirely, this PR looks good

Reply:
done